stacker_blueprints/firehose/base.py

```diff
@@ -133,68 +133,65 @@ def kms_key_policy(key_use_arns, key_admin_arns):
             Resource=["*"]
         )
     )
-    if key_use_arns:
```

Comment:
Why get rid of the if statements?  This makes it so if you don't pass in any key_use_arns (ie: its just `[]`) it doesn't worry about creating the statements (which would have a blank Principal field - and may break).